app/views/requests.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Requests</h1>
+<ul>
```

Comment:
I think it will make sense to add a filter button later that allows you to remove pending or confirmed requests. Can we put this as a nice to have/later user story?